StripeCore/StripeCore/Source/Analytics/STPAnalyticEvent.swift

```diff
@@ -77,7 +77,21 @@ import Foundation
 
     // MARK: - Embedded Payment Element init
     case mcInitEmbedded = "mc_embedded_init"
-
+    
+    // MARK: - Embedded Payment Element loading
+    case mcLoadStartedEmbedded = "mc_load_started_embedded"
+    case mcLoadSucceededEmbedded = "mc_load_succeeded_embedded"
+    case mcLoadFailedEmbedded = "mc_load_failed_embedded"
```

Comment:
Let's reuse `mc_load_started` etc and send a new param that says what type of integration it is (PS vs FC vs Embedded).